doc/sphinx/source/graph-basics.rst

```diff
@@ -1,25 +1,93 @@
 .. graph-basics:
 
+#############
 Graph Basics
-============
+#############
 
-This section describes the basic concepts you need to know when 
-constructing a graph.
 
+This section explains some of the primary concepts used in the nGraph Library.
+Some new ideas regarding our unique departure from the first generation of deep 
+learning software design are also introduced. 
 
-Framework Bridges
-------------------
+Many frameworks have a baked-in assumption that performant deep learning systems 
+require stacks of GPUs. Given this assumption, the frameworks tend to yield best 
+performance `only when given full control`_ of the GPU devices, and if 
```

Comment:
I don't think these are good sources to cite.

Reply:
removed